src/app/components/add-board-modal/add-board-modal.component.ts

```diff
@@ -56,13 +61,36 @@ export class AddBoardModalComponent implements OnInit {
     this.tags = this.tags.filter(tag => tag != tagRemove)
   }
 
-  handleImageUpload(e) {
-    var file = e.target.files[0];
-    var reader = new FileReader();
-    reader.onload = (f) => {
-        this.bgImgURL = f.target?.result;
-    };
-    reader.readAsDataURL(file);
+  compressFile() {
+    // similar to code in configuration modal
+    // might refactor in future to avoid copy paste
```

Comment:
You commented it here and forgot perhaps, but I think having it refactored would be nice.